### PR TITLE
Fix for #10

### PR DESCRIPTION
**Pull request by m-a-r-c-e-l-i-n-o**
Fix for #10

### CODE REVIEW

**Comment by internettrans** [comment]
I'll take a look at this later on today -- thanks again for your help.

**Comment by m-a-r-c-e-l-i-n-o** [comment]
@joeldenning Thank you. Learned a lot in the last few days.

File: README.md [hunk 1]
```diff
@@ -52,27 +69,41 @@ This will run your jasmine tests, loading all the tests with JSPM instead of nod
...
+  // Defaults to path where `jspmjasmine` is being executed
+  packagePath: "RELATIVE/PATH/TO/PACKAGE/JSON/DIRECTORY/",
+
+  // Add a coverage report
+  // Defaults to undefined
+  coverage: {
+
+    // Provide a custom directory for storing coverage reports
+    // Defaults to "./coverage/"
+    coverageDir: "RELATIVE/PATH/TO/COVERAGE/DIRECTORY/",
```
Comment (internettrans) [inline README.md]: Shouldn't this be just `dir`? See https://github.com/CanopyTax/node-jspm-jasmine/blob/master/src/node-jspm-jasmine.js#L54

File: README.md [hunk 2]
```diff
@@ -52,27 +69,41 @@ This will run your jasmine tests, loading all the tests with JSPM instead of nod
...
+
+    // Provide a custom type for coverage reports
+    // Defaults to "html" (all report types shipped with Istanbul are supported)
+    coverageReporter: "TYPE",
```
Comment (internettrans) [inline README.md]: Same here -- just `reporter`

File: README.md [hunk 3]
```diff
@@ -52,27 +69,41 @@ This will run your jasmine tests, loading all the tests with JSPM instead of nod
...
+
+    // Provide files to be instrumented by Istanbul coverage
+    // Defaults empty array (no files)
+    coverageFiles: "RELATIVE/GLOB/PATH/TO/FILES/**/*.js"
```
Comment (internettrans) [inline README.md]: and same here. `files`

File: README.md [hunk 4]
```diff
@@ -52,27 +69,41 @@ This will run your jasmine tests, loading all the tests with JSPM instead of nod
...
+  },
+
+  // Allow `node-jspm-jasmine` to empty coverage directory before creating a new report
+  // Defaults to undefined (no clean up)
+  cleanCoverageDir: true
```
Comment (internettrans) [inline README.md]: And this one is `cleanDir`

**Comment by m-a-r-c-e-l-i-n-o** [comment]
Okay, I think that fixes it :)

File: README.md [hunk 5]
```diff
@@ -52,27 +69,41 @@ This will run your jasmine tests, loading all the tests with JSPM instead of nod
...
+    reporter: "TYPE",
+
+    // Provide files to be instrumented by Istanbul coverage
+    // Defaults empty array (no files)
+    files: "RELATIVE/GLOB/PATH/TO/FILES/**/*.js"
+  },
+
+  // Allow `node-jspm-jasmine` to empty coverage directory before creating a new report
+  // Defaults to undefined (no clean up)
+  cleanDir: true
```
Comment (internettrans) [inline README.md]: This should be inside of the coverage object. https://github.com/CanopyTax/node-jspm-jasmine/blob/master/src/node-jspm-jasmine.js#L56

**Comment by m-a-r-c-e-l-i-n-o** [comment]
I'm really on a roll today, aren't I? lol.

**Pull request merged**